### PR TITLE
Update guide

### PR DESCRIPTION
**Pull request by WinterSilence**

### CODE REVIEW

File: guide/orm/examples/validation.md [hunk 1]
```diff
@@ -54,46 +54,43 @@ This example will create user accounts and demonstrate how to handle model and c
 	}
 
 ## HTML Form
+	<?php echo Form::open(Request::$current->url()) ?>
+		<?php echo Form::label('username', 'Username') ?>
```
Comment (kemo) [inline guide/orm/examples/validation.md:58]: Shouldn't we use open_short_tag for view output?

Comment (WinterSilence) [inline guide/orm/examples/validation.md:58]: current coding style support only long open tag for compatibility with older versions PHP

Comment (kemo) [inline guide/orm/examples/validation.md:58]: I meant "short echo tags"; `<?= ?>`. These are actually recommended by recent standards (PSR-1) and permanently enabled as of 5.4. What PHP version should we aim for with Ko 3.4, anyone?

Comment (WinterSilence) [inline guide/orm/examples/validation.md:58]: I understand what you're saying, I'm also in favor of using short tags, but  need to update Kohana coding style. Also, current Kohana coding style opposite PSR style, minimal version PHP 5.3+

**Pull request merged**